packages/components/form/src/form.stories.ts

```diff
@@ -188,13 +179,40 @@ export const All: Story = {
         </sl-checkbox-group>
       </sl-form-field>
 
+      <sl-form-field hint="Hint text" label="Combobox">
+        <sl-combobox name="combobox" required>
+          <sl-listbox>
+            <sl-option>Option 1</sl-option>
+            <sl-option>Option 2</sl-option>
+            <sl-option>Option 3</sl-option>
+            <sl-option>Option 4</sl-option>
+          </sl-listbox>
+        </sl-combobox>
+      </sl-form-field>
+
       <sl-form-field hint="Hint text" label="Radio group">
         <sl-radio-group name="radioGroup" required>
           <sl-radio value="1">One</sl-radio>
           <sl-radio value="2">Two</sl-radio>
           <sl-radio value="3">Three</sl-radio>
         </sl-radio-group>
       </sl-form-field>
+
+      <sl-form-field hint="Hint text" label="Search field">
```

Comment:
why would you have a search field inside a form? Or at least this form... I think the use case is too different to just put it in here